include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -616,15 +661,20 @@ struct __parallel_radix_sort_iteration
 
         // 1. Count Phase
         sycl::event __count_event = __radix_sort_count_submit<_RadixCountKernel, __radix_bits, __is_ascending>(
-            __exec, __segments, __block_size, __radix_offset, __in_rng, __tmp_buf, __dependency_event, __proj
+            __exec, __segments, __block_size, __radix_offset, __in_rng, __tmp_buf,
+            __dependency_event, __proj
 #if _ONEDPL_COMPILE_KERNEL
             , __count_kernel
 #endif
         );
 
+        //A flag to specify "no re-order execution" if data are the same on the current sort iteration.
+        uint16_t __no_op = 0;
```

Comment:
Let's make it aligned with the rest of the file using fixed-width integral types from `::std` namespace. Also, should the type be changed to `::std::uint8_t` or `bool`?

Reply:
probably, it makes sense